### PR TITLE
Tests for TracepointInfoManager

### PR DESCRIPTION
**Pull request by MariaSandru27**
Tests for TracepointInfoManager
Removed unused functions in TracepointEventBuffer

### CODE REVIEW

File: OrbitCore/CMakeLists.txt [hunk 1]
```diff
@@ -94,7 +94,7 @@ target_sources(OrbitCoreTests PRIVATE
     StringManagerTest.cpp
     SymbolHelperTest.cpp
     UtilsTest.cpp
-)
+    TracepointInfoManagerTest.cpp)
```
Comment (danielfenner) [inline OrbitCore/CMakeLists.txt:97]: Should be sorted alphabetically.

File: OrbitCore/TracepointInfoManagerTest.cpp [hunk 2]
```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
```
Comment (danielfenner) [inline OrbitCore/TracepointInfoManagerTest.cpp:5]: "" instead of <>

**Review by danielfenner — APPROVED** [review]
Please fix trivial comments - lgtm otherwise!

File: OrbitCore/CMakeLists.txt (hunk 1 above)
Comment (MariaSandru27) [reply OrbitCore/CMakeLists.txt]: Thank you!

File: OrbitCore/TracepointInfoManagerTest.cpp (hunk 2 above)
Comment (MariaSandru27) [reply OrbitCore/TracepointInfoManagerTest.cpp]: Thank you!

**Comment by MariaSandru27** [comment]
> Please fix trivial comments - lgtm otherwise!

Done. Thank you a lot for taking the time!

**Review by dimitry- — APPROVED** [review]

**Pull request merged**